style.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-gap: 10px;
+  background-color: #e94a4a;
+  padding: 10px;
+}
+
+
+
+
+
+.grid-container {
+  display: grid;
+  width: 375px;
+  height: 2500px;
+  grid-template-rows: auto auto auto auto;
+  justify-content: center;
+  align-content: center;
+  grid-gap: 10px;
+  background-color: #e5e5e5;
+  padding: 20px;
+  margin: 10px;
+  font-size: 30px;
+  justify-content: center;
+  align-content: center;
+}
+
+.flex1 
+  display: flex;
+  width: 327px;
+  height: 552px;
+  margin: 16px;
+  top: 114px;
+  left: 24px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  background-color: #fff;
+  flex-direction: column;
+  }
+
+.snapshoot {
+  display: flex
+  width: 295px;
+  height: 220px;
+  justify-content: center
+  padding: 50px 36px 15px 24px;
+}
+
+
+#Tonic {
+  display: flex;
+  width: 295px; hjik
+  height: 36;
```

Comment:
- [ ] Please be more specific in your comments and try to tell the problem more elaborately so that the student should be able to understand what are you trying to say with a solid reason to make a student a better developer. 

### Your Comment
> Fantastic code. Whereas, is important to delete unnecessary characters: